test/converters/converters_test.go

```diff
@@ -46,98 +46,98 @@ func TestXCresult3Converters(t *testing.T) {
 				Tests:    15,
 				Failures: 3,
 				Errors:   0,
-				Time:     0.7596049308776855,
+				Time:     0.759,
 				TestCases: []junit.TestCase{
 					// class rtgtrghtrg3UITests: XCTestCase inside rtgtrghtrgUITests class
 					{
 						Name:      "testExample",
 						ClassName: "_TtCC17rtgtrghtrgUITests17rtgtrghtrgUITests18rtgtrghtrg3UITests",
-						Time:      0.031695008277893066,
+						Time:      0.032,
 					},
 					{
 						Name:      "testFailMe",
 						ClassName: "_TtCC17rtgtrghtrgUITests17rtgtrghtrgUITests18rtgtrghtrg3UITests",
-						Time:      0.09049093723297119,
+						Time:      0.09,
 						Failure: &junit.Failure{
-							Value: "/Users/tamaspapik/Develop/ios/rtgtrghtrg/rtgtrghtrgUITests/rtgtrghtrgUITests.swift:68 - XCTAssertTrue failed",
```

Comment:
the new parsing method removes the failing line info. can we get this back from somewhere?

Reply:
I checked the this with more Xcode 16 xcresult files and we do have the file and line number in the error result:
```
DarkAndLightModeTests.swift:25: failed - This test failed
```
And actually we a test case which showcases this:
https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/620564537af982a2638209977be29473d5d29a57/test/converters/xcresult3/converter_test.go#L142

Apart from these I did not find any other way with the new way to get more error details.

What I suspect is that the xcresult file this test uses has an older data structure which maybe the new extraction method cannot fully understand. And could explain why it always works with Xcode 16 generated ones.